src/commentsDashboard/index.jsx

```diff
@@ -0,0 +1,98 @@
+import PostsList from "../postsList/";
+import { useEffect, useState } from "react";
+import { Button, ButtonGroup } from "@mui/material";
+import PhotosList from "../photosList/index.jsx";
+
+const DisplayMode = {
+  posts: "Browse posts:",
+  photos: "Browse photos:",
+};
+
+const useFetchedUsers = () => {
+  const [users, setUsers] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    fetch("https://jsonplaceholder.typicode.com/users")
+      .then((response) => response.json())
+      .then((data) => setUsers(data))
+      .catch((error) => {
+        console.log("error fetching data", error);
+      })
+      .finally(() => setIsLoading(false));
+  }, []);
+  return { users, isLoading };
+};
+
+const CommentsDashboard = () => {
+  const { users, isLoading } = useFetchedUsers();
+  const [activeUser, setActiveUser] = useState(null);
+  const [displayMode, setDisplayMode] = useState(DisplayMode.posts);
+
+  const newActiveUser = (userId) => {
```

Comment:
This function name sound like a variable holding some user object. Please propose a better name.

Reply:
done